packages/protocol/contracts/governance/EpochRewards.sol

```diff
@@ -382,20 +420,25 @@ contract EpochRewards is Ownable, Initializable, UsingPrecompiles, UsingRegistry
 
   /**
    * @notice Calculates the per validator epoch payment and the total rewards to voters.
-   * @return The per validator epoch payment and the total rewards to voters.
+   * @return The per validator epoch reward, the total rewards to voters, and the total community
+   * reward
    */
-  function calculateTargetEpochPaymentAndRewards() external view returns (uint256, uint256) {
+  function calculateTargetEpochRewards() external view returns (uint256, uint256, uint256) {
     if (frozen) {
-      return (0, 0);
+      return (0, 0, 0);
     }
 
-    uint256 targetEpochRewards = getTargetEpochRewards();
-    uint256 targetTotalEpochPaymentsInGold = getTargetTotalEpochPaymentsInGold();
-    uint256 targetGoldSupplyIncrease = targetEpochRewards.add(targetTotalEpochPaymentsInGold);
+    uint256 targetVoterReward = getTargetEpochRewards();
+    uint256 targetGoldSupplyIncrease = _getTargetGoldSupplyIncrease();
     FixidityLib.Fraction memory rewardsMultiplier = _getRewardsMultiplier(targetGoldSupplyIncrease);
     return (
       FixidityLib.newFixed(targetValidatorEpochPayment).multiply(rewardsMultiplier).fromFixed(),
-      FixidityLib.newFixed(targetEpochRewards).multiply(rewardsMultiplier).fromFixed()
+      FixidityLib.newFixed(targetVoterReward).multiply(rewardsMultiplier).fromFixed(),
+      FixidityLib
```

Comment:
meganit: For readability consider `uint256 targetCommunityReward = FixidityLib.newFixed(targetGoldSupplyIncrease).multiply(communityRewardFraction);` above